plugins/scaffolder-backend/src/lib/templating/SecureTemplater.ts

```diff
@@ -46,26 +45,14 @@ const { render, renderCompat } = (() => {
   });
   compatEnv.addFilter('jsonify', compatEnv.getFilter('dump'));
 
-  if (typeof parseRepoUrl !== 'undefined') {
-    const safeHelperRef = parseRepoUrl;
-
-    env.addFilter('parseRepoUrl', repoUrl => {
-      return JSON.parse(safeHelperRef(repoUrl))
-    });
-    env.addFilter('projectSlug', repoUrl => {
-      const { owner, repo } = JSON.parse(safeHelperRef(repoUrl));
-      return owner + '/' + repo;
-    });
-  }
-
-  if (typeof additionalTemplateFilters !== 'undefined') {
-    for (const [filterName, filterFn] of Object.entries(additionalTemplateFilters)) {
+  if (typeof templateFilters !== 'undefined') {
+    for (const [filterName, filterFn] of Object.entries(templateFilters)) {
       env.addFilter(filterName, (...args) => JSON.parse(filterFn(...args)));
     }
   }
 
-  if (typeof additionalTemplateGlobals !== 'undefined') {
-    for (const [globalName, global] of Object.entries(additionalTemplateGlobals)) {
+  if (typeof templateGlobals !== 'undefined') {
+    for (const [globalName, global] of Object.entries(templateGlobals)) {
```

Comment:
Is it perhaps best to not name this one specifically `global` in a node context? :)